src/components/ExtendedFooter.tsx

```diff
@@ -0,0 +1,125 @@
+import {
+  RiFacebookCircleFill,
+  RiLinkedinBoxFill,
+  RiInstagramFill,
+  RiGithubFill,
+  RiDiscordFill,
+} from "react-icons/ri";
+import logoIcon from "../assets/imgs/logo-icon-name-h.svg";
+
+export function ExtendedFooter() {
+  return (
+    <footer className="border-t-2 border-[#E0EDFD]">
+      <div className="flex my-14 ml-14 bg-gray-light">
+        <div className="mr-14">
+          <img src={logoIcon} width="265px" />
+          <h2 className="font-semibold text-gray-darker text-[25px] mt-2 ">
```

Comment:
Remover o espaçamento após a última classe
```suggestion
          <h2 className="font-semibold text-gray-darker text-[25px] mt-2">
```